ams-backend.ts

```diff
@@ -1,16 +1,53 @@
 import cors from 'cors'
+import dayjs from 'dayjs'
 import express from 'express'
+import fs from 'fs'
 import morgan from 'morgan'
-import dayjs from 'dayjs'
+import path from 'path'
 import 'dayjs/locale/ja'
 import * as accesslogsRoutes from './app/routes/accessLogsRoutes'
 import * as inRoomUsersRoutes from './app/routes/inRoomUsersRoutes'
 import * as readerInputRoutes from './app/routes/readerInputRoutes'
 import * as sseRoutes from './app/routes/sseRoutes'
 import { amsOptions } from './config'
 
+function prepareMorgan () {
+  // morganの:dateはUTCなので、日本時間を表示するものを作る
+  morgan.token('localdate', () => dayjs().locale('ja').format())
+
+  // ログを吐くフォルダを作る
+  try {
+    fs.mkdirSync(amsOptions.logPath)
+  } catch (err) {
+    if (err.code === 'EEXIST') {
+      // EEXISTの場合は既にフォルダがあるので無視する
+    } else {
+      // それ以外の場合はエラーを出力して死ぬ
+      console.error('Could not create folder:', err)
+      process.exit(1)
+    }
+  }
```

Comment:
超細かいですが，existsSyncで先に存在確認をすれば発生すると分かっている`EEXIST`を発生させるのを回避できて良いんじゃないかなと思いました．完全に修正前のコードと挙動が同じ自信はありませんが，少なくとも手元で動かしてみた感じ同じように見えました．議論の余地ありですけどね～．
```suggestion
if (!fs.existsSync(amsOptions.logPath)) {
    try {
      fs.mkdirSync(amsOptions.logPath)
    } catch (err) {
      // ファイルが存在している以外の場合はエラーを出力して死ぬ
      console.error('Could not create folder:', err)
      process.exit(1)
    }
  }
```

Reply:
これも物凄〜く細かい話になるんですが、可能性としては`existsSync`を呼んでから`mkdirSync`を呼ぶまでの間に対象のディレクトリが作られちゃうシチュエーションが存在しうるんです。
なので大きなインセンティブがない限りは現状のコードで行きたいと思います。

Reply:
あ～ファイル操作系の関数を呼ぶのは一回限りの方が良いってことか．OKです．現状のコードで行きましょう．